admin_guide/persistent_storage_nfs.adoc

```diff
@@ -6,52 +6,34 @@
 :experimental:
 :toc: macro
 :toc-title:
+:prewrap!:
 
 toc::[]
 
 == Overview
-This topic describes how to provision an OpenShift cluster with persistent storage using NFS.  Some familiarity with Kubernetes and NFS is assumed.
+You can provision your OpenShift cluster with persistent storage using NFS. Some
+familiarity with Kubernetes and NFS is assumed.
 
-The Kubernetes
-link:../dev_guide/volumes.html[Persistent Volume] framework allows
-administrators to provision a cluster with persistent storage and gives users a way to request those resources without having
-any knowledge of the underlying infrastructure.
-
-IMPORTANT: High-availability of a storage in the infrastructure is left to the underlying storage provider.
+The Kubernetes link:../dev_guide/volumes.html[Persistent Volume] framework
+allows administrators to provision a cluster with persistent storage and gives
+users a way to request those resources without having any knowledge of the
+underlying infrastructure.
```

Comment:
OK. The distinction is that the end user (developer) doesn't need to know about the underlying infrastructure. The admin would. This could be clarified.